bedrock/products/urls.py

```diff
@@ -28,6 +29,13 @@
     page('vpn/mobile/android', 'products/vpn/platforms/android.html',
          ftl_files=['products/vpn/platforms/android', 'products/vpn/shared']),
 
+    url(r'^products/vpn/$',
```

Comment:
@reemhamz no need to define a new URL here, that's already covered up on line 12 above.

Instead you'll need to add logic to the [existing view](https://github.com/mozilla/bedrock/blob/master/bedrock/products/views.py#L24) to check for experiment params and serve the right template. Let me know if you'd like to walk through doing this 👍